CMakeLists.txt

```diff
@@ -1,198 +1,247 @@
-cmake_minimum_required(VERSION 2.8.6)
+cmake_minimum_required(VERSION 3.5)
 project(rviz_visual_tools)
 
-if(NOT MSVC)
-  add_compile_options(-std=c++11)
-  # Warnings
-  add_definitions(-W -Wall -Wextra
-    #-Wcast-qual
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-W -Wall -Wextra
     -Wwrite-strings -Wunreachable-code -Wpointer-arith
     -Winit-self -Wredundant-decls
     -Wno-unused-parameter -Wno-unused-function)
 endif()
 
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+# set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
-find_package(catkin REQUIRED COMPONENTS
-  eigen_conversions
-  eigen_stl_containers
-  geometry_msgs
-  graph_msgs
-  shape_msgs
-  roscpp
-  roslint
-  rostest
-  rviz
-  sensor_msgs
-  std_msgs
-  tf_conversions
-  trajectory_msgs
-  visualization_msgs
-)
+# TODO(mlautman): find alternatives to missing pkgs
+# find_package(eigen_conversions)
+# find_package(tf_conversions)
+# find_package(graph_msgs REQUIRED)
```

Comment:
graph_msgs is maintained by PickNik and just needs to be released to ROS2, but its an odd feature that i agree can be skipped for now